snap/local/etc/my.cnf

```
@@ -0,0 +1,8 @@
+# Default server settings
+[mysqld]
+user                = snap_daemon
+bind-address        = 127.0.0.1
+mysqlx-bind-address = 127.0.0.1
+
+# Import all .cnf files from configuration directory
+!includedir /var/snap/mysql/current/etc
```

Should configuration directory follow mysql standard path on `/etc`

```suggestion
!includedir /var/snap/mysql/current/etc/mysql/
```

---

But thinking about it, this setup is much simpler.